packages/banner/src/BannerAnimator/BannerAnimator.js

```diff
@@ -128,23 +128,35 @@ const BannerAnimator = props => {
     [props]
   );
 
+  function usePreviousStatus(value){
+    const ref = useRef(null);
+    useEffect(() => {
+      ref.current = value;
+    });
+    return ref.current;
+  }
+
+  const prevStatus = usePreviousStatus(status);
+
   useEffect(
     () => {
-      const { status: prevStatus } = props;
+      const expandStatuses = prevStatus === statuses.COLLAPSED && status === statuses.EXPANDING ||
```

Comment:
Replace all the condition using this : 
` const expandStatuses =
        (prevStatus === statuses.COLLAPSED && status === statuses.EXPANDING) ||
        (prevStatus === statuses.COLLAPSING && status === statuses.EXPANDING);
`